src/melee.cpp

```diff
@@ -1870,11 +1870,11 @@ std::vector<special_attack> player::mutation_attacks(Creature &t) const
             }
 
             ///\EFFECT_UNARMED increases chance of attacking with mutated body parts
-
             ///\EFFECT_DEX increases chance of attacking with mutated body parts
-            const int chance = std::max( 1, mut_atk.chance - get_dex() - unarmed );
+            const float chance = get_dex() + unarmed / mut_atk.chance;
             add_msg( m_debug, "%s proc chance: %d", pr.first.c_str(), chance );
```

Comment:
It would be better to print both sides of `x_in_y`:
```
add_msg( m_debug, "%s proc chance: %.1f in %.1f", pr.first.c_str(), get_dex() + unarmed, mut_atk.chance );
```